Gemfile.lock

```diff
@@ -509,8 +501,6 @@ PLATFORMS
   arm64-darwin-21
   ruby
   x64-mingw-ucrt
-  x64-mingw32
-  x86-mingw32
```

Comment:
This is what's causing the deletion of all the native platforms. Resolution gives up and falls back to ruby-only group. See: https://github.com/rubygems/rubygems/issues/8505#issuecomment-2667723563